### PR TITLE
Updated the GKE version and syntax yaml files

### PR DESCRIPTION
**Pull request by anirudh-hegde**
**Notes for Reviewers**

This PR fixes syntax of YAML files and updates GKE version

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .goreleaser.yml
Comment (leecalcote) [inline .goreleaser.yml:1]: @anirudh-hegde thanks. If there's no meaningful change other than whitespace, let's forego inclusion of this file / these changes in the PR.

File: mesheryctl/pkg/utils/fixtures/TestConfig.yaml [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+    version: latest
   local:
     endpoint: http://localhost:9081
     token: default
     platform: docker
     components:
-      - meshery-istio
+    - meshery-istio
     channel: stable
     version: v0.5.10
```
Comment (leecalcote) [inline mesheryctl/pkg/utils/fixtures/TestConfig.yaml:17]: @asubedy, if we don't use a specific version number for each test of the stable release of an adapter, will this make sense in the [Compatibility Matrix](https://docs.meshery.io/installation/compatibility-matrix)? I suspect not.

File: mesheryctl/pkg/utils/fixtures/platform/original/TestChangeEndpointConfig.yaml [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+    version: latest
   local:
     endpoint: http://localhost:9081
     token: default
     platform: docker
     components:
-      - meshery-istio
+    - meshery-istio
     channel: stable
     version: v0.5.10
```
Comment (leecalcote) [inline mesheryctl/pkg/utils/fixtures/platform/original/TestChangeEndpointConfig.yaml:17]: perhaps, we're in need of identifying the last stable release's version number. In the case of meshery-istio, that currently would be v0.7.0.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: .goreleaser.yml
Comment (anirudh-hegde) [reply .goreleaser.yml:1]: On VSCode, accidentaly I pressed Ctrl+Alt+L to this file

**Comment by leecalcote** [comment]
Reviewing again. No changes since last feedback offered.

**Comment by anirudh-hegde** [comment]
GKE versions : 1.29.1-gke.1575000
If required please refer: https://cloud.google.com/kubernetes-engine/docs/release-notes

File: mesheryctl/pkg/utils/fixtures/TestConfig.yaml (hunk 1 above)
Comment (leecalcote) [reply mesheryctl/pkg/utils/fixtures/TestConfig.yaml:17]: @asubedy?

File: .goreleaser.yml
Comment (leecalcote) [reply .goreleaser.yml:1]: @anirudh-hegde, are you still working on fixing this?

**Comment by leecalcote** [comment]
PR abandoned.

**Pull request closed without merging**